optax/_src/linear_algebra.py

```diff
@@ -30,10 +32,10 @@ def global_norm(updates: base.Updates) -> base.Updates:
 
 
 def power_iteration(
-    matrix,
-    num_iters=100,
-    error_tolerance=1e-6,
-    precision=lax.Precision.HIGHEST):
+    matrix: chex.Array,
+    num_iters: Optional[int]=100,
+    error_tolerance: Optional[float]=1e-6,
```

Comment:
As above and for the other types you have marked as Optional too - please check whether it would be possible/intended to pass `None`.